### PR TITLE
Init data_directory on runtime upgrade

### PR DESCRIPTION
**Pull request by iorveth**
Closes https://github.com/Joystream/joystream/issues/2171

### CODE REVIEW

File: cli/src/Api.ts [hunk 1]
```diff
@@ -527,8 +527,8 @@ export default class Api {
     return exists ? await this._api.query.contentDirectory.entityById<Entity>(id) : null
   }
 
-  async dataObjectByContentId(contentId: ContentId): Promise<DataObject | null> {
```
Comment (mnaamani) [inline cli/src/Api.ts:530]: I think you in-inadvertently auto-replaced occurrences in client code. The queries will start with a lower case.
Perhaps revert these and leave it for another PR.

**Review by mnaamani — CHANGES_REQUESTED** [review]
Implementation looks good.
Just small issue, with name change in client packages.

File: cli/src/Api.ts (hunk 1 above)
Comment (iorveth) [reply cli/src/Api.ts:530]: Thank you for pointing that. I hope it wouldn`t be hard to make them lower case again

Comment (iorveth) [reply cli/src/Api.ts:530]: Done https://github.com/Joystream/joystream/pull/2173/commits/7f1cd93229fb8188bc4055d5579ea9b2c37c7fcc

**Review by mnaamani — APPROVED** [review]
The `types/augment-*` files need to be generated separately but that can be fixed later
